pkg/webhook/servicecatalog/clusterservicebroker/validation/handler_access_test.go

```diff
@@ -0,0 +1,290 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation_test
+
+import (
+	"context"
+	"errors"
+	sc "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/kubernetes-incubator/service-catalog/pkg/webhook/servicecatalog/clusterservicebroker/validation"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	"k8s.io/api/authorization/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"testing"
+)
+
+const (
+	AllowedSecretName = "csb-secret-name"
+	DeniedSecretName  = "denied-csb-secret-name"
+)
+
+// Reactors are not implemented in 'sigs.k8s.io/controller-runtime/pkg/client/fake' package
+// https://github.com/kubernetes-sigs/controller-runtime/issues/72
+// instead it is used custom client with override Create method
+type fakedClient struct {
+	client.Client
+}
+
+// Create overrides real client Create method for the test
+func (m *fakedClient) Create(ctx context.Context, obj runtime.Object, opts ...client.CreateOptionFunc) error {
+	if _, ok := obj.(*v1.SubjectAccessReview); !ok {
+		return errors.New("Input object is not SubjectAccessReview type")
+	}
+
+	if obj.(*v1.SubjectAccessReview).Spec.ResourceAttributes.Name == AllowedSecretName {
+		obj.(*v1.SubjectAccessReview).Status.Allowed = true
+	}
+
+	return nil
+}
+
+func TestAdmissionHandlerAccessToBrokerAllowed(t *testing.T) {
+	//Given
+	err := sc.AddToScheme(scheme.Scheme)
+	require.NoError(t, err)
+
+	request := admission.Request{
+		AdmissionRequest: admissionv1beta1.AdmissionRequest{
+			UID:       "3333-cccc",
+			Name:      "test-broker",
+			Namespace: "test-handler",
+			Kind: metav1.GroupVersionKind{
+				Kind:    "ClusterServiceBroker",
+				Version: "v1beta1",
+				Group:   "servicecatalog.k8s.io",
+			},
+			Object: runtime.RawExtension{},
+		},
+	}
+
+	decoder, err := admission.NewDecoder(scheme.Scheme)
+	require.NoError(t, err)
+
+	tests := map[string]struct {
+		operation admissionv1beta1.Operation
+		object    []byte
+	}{
+		"Request for Create ClusterServiceBroker should be allowed": {
```

Comment:
```suggestion
		"Request for Create ClusterServiceBroker without AuthInfo should be allowed": {
```
right now you missing why it should be allowed, same suggestion for the case below